### DISCUSSION TITLE
Pull and prune branches

### DISCUSSION THREAD

**Discussion by aminnairi** [NeogitOrg/neogit#1111] [Q&A] [answered]
When one has finished working on a branch on GitHub for instance and merged the relevant commits from one branch to the master branch, and the working branch gets deleted, it would be nice to be able to pull the changes from GitHub in order to prune branches out.

Is there a way to do this using Neogit? I'm not seing an option to prune while doing a pull, or maybe this option has been set elsewhere?

**Comment by CKolkey** [comment, marked as answer]
The `--prune` flag is available fia the `fetch` popup - recall that a `pull` is just a `fetch` plus a `merge` (or rebase). Is that what you're looking for?
